packages/marshal/src/rankOrder.js

```diff
@@ -142,16 +142,20 @@ export const makeComparatorKit = (compareRemotables = (_x, _y) => 0) => {
       case 'string': {
         // Within each of these passStyles, the rank ordering agrees with
         // JavaScript's relational operators `<` and `>`.
+        // @ts-expect-error FIXME narrowed
         if (left < right) {
           return -1;
         } else {
+          // @ts-expect-error FIXME narrowed
```

Comment:
👆 Is the intention to address all FIXMEs before landing? These could be links to an issue, otherwise.

Reply:
not before landing. maybe never. I've taken out the FIXME